### ISSUE TITLE
Talkback should only read active story page content.

### ISSUE THREAD

**Issue by gmajoulet** [ampproject/amphtml#28421]
Talkback should only read active story page content.

Talkback reads the active story page content, and then proceeds with the following page before user navigation happens.

cc @ampproject/wg-stories 
b/155936295

**Comment by cramforce** [comment]
I tried to prevent this by making every slide except the current slide `aria-hidden="true"`. If that isn't working, what can we do?

**Comment by gmajoulet** [comment]
There is a chance that your changes (https://github.com/ampproject/amphtml/pull/27820 for reference) were not in production at the time of the accessibility review.
If `aria-hidden` attributes are not enough, @processprocess found out that `visibility: hidden;` were working. That'd be a mobile only solution though.

**Comment by processprocess** [comment]
I'm unable to reproduce this bug on my Pixel 3. I wonder if it has been fixied. 

`aria-hidden` should be enough. I think the `visibility: hidden` attributes should just be used for interactive UI elements.

**Comment by gmajoulet** [comment]
Awesome, thanks to both of you 🏅
